go2html_test.go

```diff
@@ -103,10 +103,10 @@ var _ = Describe("go2html", func() {
 							map[string]interface{}{
 								"text1":             "Inserted text here.",
 								"paragraph1":        "Inserted <b>paragraph1</b> text.",
-								"div1-attr":         "title=\"Some title\"",
+								"div1-attr":         map[string]string{"title": "\"Some title\""},
```

Comment:
here we should avoid formatting for values (I mean `\"...\"`), the value string should be an exact value that will be placed later within `<attr>="<value>"`

Reply:
Fixed, as said above in the first changes request.